From 61947569a1945bb2f971dcdcaea56a94539a7eb9 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 16:02:24 +1100
Subject: [PATCH] distro-angstrom: disable psplash from angstrom includes for openmoko

this removes psplash if the "screen" property of a machine is set for
openmoko. the openmoko distribution/image explicitly provides its own splash
universally, so this is not needed. interestingly the openmoko hardware does
not say it provides a screen as a feature.
---
 conf/distro/include/angstrom-2007-for-openmoko.inc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/conf/distro/include/angstrom-2007-for-openmoko.inc b/conf/distro/include/angstrom-2007-for-openmoko.inc
index 0c7fe5b..669e2aa 100644
--- a/conf/distro/include/angstrom-2007-for-openmoko.inc
+++ b/conf/distro/include/angstrom-2007-for-openmoko.inc
@@ -218,13 +218,13 @@ DISTRO_EXTRA_DEPENDS += " \
 
 DISTRO_EXTRA_RDEPENDS += "\
     update-modules \
-    ${@base_contains("MACHINE_FEATURES", "screen", "psplash", "",d)} \   
     angstrom-version \
     opkg opkg-collateral distro-feed-configs \
     util-linux-mount util-linux-umount \
     ${EXTRA_STUFF} \
     ${DEBUG_APPS} \
     "
+#    ${@base_contains("MACHINE_FEATURES", "screen", "psplash", "",d)}
 
 DISTRO_EXTRA_RRECOMMENDS += " \
     kernel-module-vfat \
-- 
1.5.4.3

